charts/newrelic-infrastructure/values.yaml

```
@@ -84,6 +84,9 @@ kubelet:
     - operator: "Exists"
       effect: "NoExecute"
   nodeSelector: {}
+  # -- (bool) Sets pod's hostNetwork. Uncomment to configure hostnetwork to ksm deployment only, bypasses global/common variable 
+  # @default ==  `global.hostNetwork`
+  #hostNetwork: false
```

I think having the value commented out will cause [helm-docs](https://github.com/norwoodj/helm-docs) to ignore it. Can you verify this?
```suggestion
  # -- (bool) Sets pod's hostNetwork. Uncomment to configure hostnetwork to ksm deployment only, bypasses global/common variable 
  # @default -- Same as `global.hostNetwork`
  hostNetwork:
```

---

done